doc/tut1.rst

```diff
@@ -644,9 +651,9 @@ the exit.
         return
       result = result + i
 
-  echo sumTillNegative() # echos 0
-  echo sumTillNegative(3, 4, 5) # echos 12
-  echo sumTillNegative(3, 4 , -1 , 6) # echos 7
+  echo sumTillNegative() # echoes 0
```

Comment:
```suggestion
  assert sumTillNegative() == 0
```
simpler + more idiomatic, ditto below

Reply:
But then the reader must already know about `assert`.

Reply:
assert is in virtually all programming languages though, it's reasonable to expect users would get its meaning (but the tutorial can add a comment on the 1st instance of `assert`)

Reply:
I would agree with this change if this were some test in the stdlib.
This being a tutorial, I prefer `echo` in all the examples where is currently.